dll/win32/msxml3r/lang/ru-RU.rc

```diff
@@ -0,0 +1,7 @@
+LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_XML_FILE "Документ XML"
+    IDS_XLS_FILE "XSL таблица стилей"
```

Comment:
```suggestion
    IDS_XLS_FILE "Таблица стилей XSL"
```

Reply:
I don't know how to get rid of the other stuff. This was back when I was getting started with GIT and made some mistakes.